src/model_repository_manager.cc

```diff
@@ -1607,6 +1658,18 @@ ModelRepositoryManager::UnloadAllModels()
   return Status::Success;
 }
 
+Status
+ModelRepositoryManager::StopAllModels()
+{
+  return model_life_cycle_->StopAllModels();
+}
+
+const std::set<std::tuple<std::string, int64_t, size_t>>
```

Comment:
The set can be made more descriptive to make it more comprehensible.

Reply:
Added comment about the return tuple in the function description, not sure if we need to `using ..` for it as it is not super nested that needs a descriptive name.